jupyterhub/values.yaml

```diff
@@ -93,11 +93,15 @@ proxy:
   https:
     enabled: true
     type: letsencrypt
+    #type: manualsecret
     letsencrypt:
       contactEmail: ''
     manual:
       key:
       cert:
+    secret: ""
+    keyname: ""
+    crtname: ""
```

Comment:
Can we restructure this a tiny bit so it looks like:

```yaml
type: "secret"
secret:
  name: ""
  keyname: ""
  crtname: ""
```

so that the secret-related config is under https.secret instead of at the top level and the 'type' key matches the sub-dict ('secret')?